data_import/admin.py

```diff
@@ -62,5 +63,23 @@ def save_model(self, request, obj, form, change):
         super().save_model(request, obj, form, change)
 
 
+class AdminStandardizedFile(admin.ModelAdmin):
+    form = UploadForm
+
+    def save_model(self, request, obj, form, change):
+        upload = Upload.objects.create()
+        standardized_file = form.cleaned_data['standardized_file']
+        reporting_year = form.cleaned_data['reporting_year']
+
+        obj.upload = upload
+        obj.standardized_file = standardized_file
+        obj.reporting_year = reporting_year
+
+        obj.copy_to_database()
```

Comment:
You'll want to save the object before calling this task, since it relies on the object having an ID.